### ISSUE TITLE
TPG EPICS/ioc remove dbLoadRecords for devTPG300.db and devTPG500.db 

### ISSUE THREAD

**Issue by zsoltkebel** [ISISComputingGroup/IBEX#7984]
It looks to be a resibual mistake from the TPG ticket: https://github.com/ISISComputingGroup/IBEX/issues/7458

These two lines remained in the st-common.cmd that were meant to be removed.

```
$(IF300) dbLoadRecords("$(TPG)/db/devTPG300.db","P=$(MYPVPREFIX)$(IOCNAME):, PORT=L0,RECSIM=$(RECSIM=0),DISABLE=$(DISABLE=0)")
$(IFNOT300) dbLoadRecords("$(TPG)/db/devTPG500.db","P=$(MYPVPREFIX)$(IOCNAME):, PORT=L0,RECSIM=$(RECSIM=0),DISABLE=$(DISABLE=0)")
```

## Requirements
- [ ] Remove [these two](https://github.com/ISISComputingGroup/EPICS-ioc/blob/master/TPG300/iocBoot/iocTPG300-IOC-01/st-common.cmd#L46L47) lines fom the st-common.cmd 
- [ ] Update all instruments that we deployed to
- [ ] Add to master
- [ ] Make notes in appropriate hotfix columns of the wiki

**Comment by zsoltkebel** [comment]
# PRs
- https://github.com/ISISComputingGroup/EPICS-ioc/pull/806

As these two extra lines don't couse an error (other than trying to load something, failing and carrying on) I remove the urgent label and think it is enough if fixed later.

Reviewer can decide if these need to be hotfixed on machines and request rework if needed.

**Comment by rerpha** [comment]
we can see which instruments are using the TPG IOC with the instrumentChecker's [get_ioc_usage.bat](https://github.com/ISISComputingGroup/InstrumentChecker/blob/master/get_ioc_usage.bat) script. we should run this before deciding which instruments to hotfix whatever the solution is for this ticket.

**Comment by zsoltkebel** [comment]
I doubt that is the case, the test cover per channel setting and reading here: so the IOC shouldn't swap any of them: https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/blob/master/common_tests/tpgx00.py#L130

Also I noticed that the presure channel he block is reading sometimes spikes up to around 9900. It could be that when I took the screenshot it was right at one of these spikes. I oinclude a screeshot of what I mean below. If that's the case I think everything is working as expected so I didn't create a ticket for that.

![image](https://github.com/ISISComputingGroup/IBEX/assets/25386639/77ced7f8-6ffa-4d0f-9963-32923be7a7ef)
